Library/Homebrew/language/python.rb

```diff
@@ -88,6 +88,17 @@ def self.setup_install_args(prefix)
       ]
     end
 
+    def self.rewrite_python_shebang(python_path)
+      Pathname(".").find do |f|
+        regex = %r{^#! ?(/usr/bin/env python([23](.[0-9])?)?|/usr/bin/python)$}
```

Comment:
```suggestion
        regex = %r{^#! ?/usr/bin/(env )?python([23](\.\d+)?)$}
```

- shorten the regex a bit
- also match `/usr/bin/python2` and `/usr/bin/python3`
- handle `python3.10` etc. when they come along